thunderhub.patch

```diff
@@ -9,15 +9,15 @@
  # ---------------
  # Install Dependencies
  # ---------------
--FROM node:14.15-alpine as deps
-+FROM arm64v8/node:14.15-alpine as deps
+-FROM node:16.15.1-alpine as deps
++FROM arm64v8/node:16.15.1-alpine as deps
```

Comment:
what's the reason for this change?

Reply:
keeping the patch in sync with upstream

Reply:
ah, so upstream updated to the `arm64v8` version of the image?

Reply:
no, the upstream version updated from node v14.15 to node v16.15.1

Reply:
I see, so why the switch to `arm64v8` then?

Reply:
Gotcha, reading patch files is confusing xD

Ok so the change from vanilla alpine to `arm64v8` alpine was already baked in.